AppCenterCrashes/AppCenterCrashesTests/MSACStackFrameTests.m

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#import "AppCenter+Internal.h"
```

Comment:
do we need this line?

Reply:
yes, it is needed for using some methods (for example from `MSACUtility` class)